pkg/network/protocols/events/consumer.go

```diff
@@ -62,13 +63,12 @@ func NewConsumer[V any](proto string, ebpf *manager.Manager, callback func([]V))
 		return nil, fmt.Errorf("unable to find map %s: %s", batchMapName, err)
 	}
 
-	eventsMapName := proto + eventsMapSuffix
-	eventsMap, found, _ := ebpf.GetMap(eventsMapName)
-	if !found {
-		return nil, fmt.Errorf("unable to find map %s", eventsMapName)
+	numCPUs, err := kernel.PossibleCPUs()
```

Comment:
I think the more "common way" in the project to get the number of CPUs is to set it `int(mapName.MaxEntries())` where `mapName` is a per-cpu array / map.
I dont have a strong opinion here, but putting that as an FYI

Reply:
this has been updated precisely because it only works with perf maps :)